.github/cloud-samples-tools/cmd/main.go

```diff
@@ -0,0 +1,149 @@
+/*
+ Copyright 2024 Google LLC
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+*/
+
+package main
+
+import (
+	"cloud-samples-tools/pkg/utils"
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+)
+
+func main() {
```

Comment:
nit: I think you can still use `flag.Parse()` here. Up to you!

Reply:
Using `flag.Parse()` made things simpler, thanks! I'm also reformatting the `printf`s to use the variadic argument functions to emulate string interpolation.